build.sbt

```diff
@@ -31,6 +24,14 @@ lazy val cassandra = project
     Dependencies.Cassandra
   )
 
+lazy val files = project
```

Comment:
The full name really stood out from the others. For example the output of `projects` command in sbt was:
```
> projects
[info] In file:/home/martynas/projects/alpakka/
[info]   * alpakka
[info]     akka-stream-alpakka-file
[info]     amqp
[info]     cassandra
[info]     docs
[info]     mqtt
```

What if we rename it to `files` everywhere?